tox.ini

```diff
@@ -69,10 +71,12 @@ commands =
 [testenv:regtests]
 description = run tests with --bigdata and --slow flags
 commands =
+    python -m pip install --upgrade pip
     pytest --bigdata --slow --basetemp={homedir}/scratch {posargs}
 
 [testenv:twine]
 description = check that the package builds sdist/wheel and that twine uploads
+changedir = {toxinidir}
```

Comment:
This should be unnecessary as there's no `changedir` in `[testenv]` to override.  I.e. this is doing nothing here.